ros/src/overhead_cv/overhead_cv/cv_recorder.py

```diff
@@ -0,0 +1,152 @@
+import math
+
+# pylint: disable=E1101
+import cv2
+
+import numpy as np
+
+import rclpy
+import rclpy.exceptions
+import rclpy.executors
+import rclpy.logging
+
+from rclpy.node import Node
+
+from std_msgs.msg import Float32MultiArray
+
+CAMERA_ID = 0
+DELAY = 1
+
+WINDOW_NAME = "OpenCV QR Code"
+
+MAP_WINDOW = "location tracking"
+
+QCD = cv2.QRCodeDetector()
+CAP = cv2.VideoCapture(CAMERA_ID)
+
+
+class CVRecorder(Node):
+    def __init__(self):
+        super().__init__("cv_recorder")
+        self.poses_emit = self.create_publisher(
+            Float32MultiArray, "multi_array_pos", 10
+        )
+
+        self.robot_points = []
+
+    # intended to convert points to width and height of sub image
+    # pylint: disable=R0201
+    def draw_robot(self, points, conversion, width, height):
+
+        max_x = max(points[1][0], points[2][0])
+        min_x = min(points[0][0], points[3][0])
+
+        max_y = max(points[2][1], points[3][1])
+        min_y = min(points[0][1], points[1][1])
+
+        point = (
+            math.floor(((max_x - (max_x - min_x) / 2) / width) * conversion[0]),
+            math.floor(((max_y - (max_y - min_y) / 2) / height) * conversion[1]),
+        )
+
+        scaled_mesh_point = np.array(
```

Comment:
`points` is already an `ndarray` so I believe you can just do this in one line. E.g., like `np.ones((3,4,2))/[3, 2] * [5,5]`